migration-table.sql

```diff
@@ -0,0 +1,7 @@
+-- migration-table.sql
+CREATE SCHEMA IF NOT EXISTS _ps_catalog;
+CREATE TABLE IF NOT EXISTS _ps_catalog.migration(
+  id TEXT NOT NULL PRIMARY KEY
```

Comment:
`id` implies an integer type to me. I'd rename to `filename` or `name`

Reply:
👍